frontend/packages/helm-plugin/integration-tests/support/commands/hooks.ts

```diff
@@ -16,6 +19,14 @@ before(() => {
   cy.login(bridgePasswordIDP, bridgePasswordUsername, bridgePasswordPassword);
   cy.document().its('readyState').should('eq', 'complete');
   // set the user settings location to local storage, so that no need of deleting config map from openshift-console-user-settings namespace
+  cy.exec(
+    `oc patch console.operator.openshift.io/cluster --type='merge' -p '{"spec":{"customization":{"perspectives":[{"id":"dev","visibility":{"state":"Enabled"}}]}}}'`,
+    { failOnNonZeroExit: false },
+  );
+  cy.reload(true);
+  cy.document().its('readyState').should('eq', 'complete');
+  guidedTour.close();
+  checkDeveloperPerspective();
```

Comment:
Let's avoid duplicate code if we can (here and below)

Reply:
Yes these will be revamped based on your point [here](https://github.com/openshift/console/pull/14684#discussion_r1922638557)